corgi/core/migrations/0067_fix_missing_go_component_type.py

```diff
@@ -0,0 +1,36 @@
+from django.db import migrations, transaction
+
+
+def fix_missing_go_component_type(apps, schema_editor):
+    """Set missing go_component_type for Golang modules
+    bundled in source RPMs, and upstream of container images"""
+    Component = apps.get_model("core", "Component")
+
+    # These are bundled into an SRPM - we assume they are always go modules
+    for component in Component.objects.filter(
+        type="GOLANG", meta_attr__go_component_type__isnull=True, meta_attr__source=["specfile"]
+    ).iterator():
+        with transaction.atomic():
+            component.meta_attr["go_component_type"] = "gomod"
```

Comment:
I'm not sure this is always the case. Perhaps an additional check on the number of '/' characters is required.